<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoRecordingDialog</class>
 <widget class="QDialog" name="VideoRecordingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLineEdit" name="fileLineEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="saveAsButton">
     <property name="text">
      <string>Save As...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="resolutionComboBox">
     <item>
      <property name="text">
       <string>Native</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Native/2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1920x1080</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1280x720</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>640x480</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Capture Mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>5</number>
      </property>
      <item>
       <widget class="QRadioButton" name="fullscreenButton">
        <property name="text">
         <string>Fullscreen</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="fullscreenNoAeroButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Fullscreen without Aero (Improves performance)</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="windowButton">
        <property name="text">
         <string>Eve window</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Encoder quality:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>5</number>
      </property>
      <item>
       <widget class="QRadioButton" name="bestButton">
        <property name="text">
         <string>Best</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="balancedButton">
        <property name="text">
         <string>Balanced</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="PerformanceButton">
        <property name="text">
         <string>Perfomance</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="cancelButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="startRecordingButton">
     <property name="text">
      <string>Start recording</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Resolution:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>VideoRecordingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
